test/yaml.test.ts

```
@@ -0,0 +1,908 @@
+        const expectedSgFilter = {
+            includeOrders: new Set([
+                "0xaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
+                "0xbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb",
+            ]),
+            excludeOrders: undefined,
+            includeOwners: new Set(["0x9999999999999999999999999999999999999999"]),
+            excludeOwners: undefined,
+            includeOrderbooks: undefined,
+            excludeOrderbooks: undefined,
+        };
+        assert.deepEqual(result.sgFilter!.includeOrders, expectedSgFilter.includeOrders);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)